### DISCUSSION TITLE
Example commands?

### DISCUSSION THREAD

**Discussion by lyonrt** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#486] [Q&A]
Can someone give some real world examples of commands I can utilize in the commands area of spa config? Just got it wired up and can do things manually now in the app!

**Comment by visualapproach** [comment]
You can schedule pretty much anything you can do manually, and some additional things. If you want to turn off the pump every day at 20:00, you can add the command+interval to do that. Or if you want to hear a beep at 06:00 - add the beep command with repeat.
